apps/src/craft/simple/craft.js

```diff
@@ -251,7 +252,7 @@ Craft.init = function (config) {
         Craft.gameController = new GameController({
           Phaser: window.Phaser,
           containerId: 'phaser-game',
-          assetRoot: Craft.skin.assetUrl(''),
+          assetRoot: Craft.skin.assetUrl('designer/'),
```

Comment:
I need to fix "Characters should have the Adventurer character skins" then my plan is to delete all non-designer Craft assets.  AFAIK the designer assets are a strict superset.